Model/AttributeSetRepository.php

```diff
@@ -23,19 +23,18 @@
 
 class AttributeSetRepository implements CodedAttributeSetRepositoryInterface
 {
-    private $attributeSetCodeRepository;
-    private $attributeGroupCodeRepository;
-    private $attributeGroupRepository;
-    /** @var AttributeSetInterfaceFactory */
-    private $attributeSetFactory;
-    private $attributeGroupFactory;
-    private $attributeManagement;
-    private $searchCriteriaBuilder;
-    private $attributeRepository;
-    private $resourceConnection;
-    private $entityTypeCodeRepository;
-    private $attributeSetRepository;
-    private $sortOrderBuilder;
+    private \SnowIO\AttributeSetCode\Model\AttributeSetCodeRepository $attributeSetCodeRepository;
```

Comment:
Again, this is really verbose when the classes are already included with `use` statements... Is this a setting somewhere in Rector?